src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -57,7 +57,7 @@ export class ScriptEvaluator {
     );
   }
 
-  static #cdpRemoteObjectToCallArgument(
+  #cdpRemoteObjectToCallArgument(
```

Comment:
many methods here can probably be top-level functions

Reply:
WIP. PTAL. Should we move `readonly #knownHandlesToRealm: Map<string, string> = new Map();` to top-level?